### PR TITLE
Challenge-2

### PR DESCRIPTION
**Pull request by RadekNowak2k19**

### CODE REVIEW

**Review by NerdbordMentor — CHANGES_REQUESTED** [review]
## Code Review Report

### Task Goal
Develop an interactive, moving news ticker according to provided assets.

### Requirements
1. Text should be moving with speed enabling reading
2. Animation should be set to linear
3. While hovering the animation should stop
4. Animation should be looped indefinitely

### Code Changes Review

#### challenge-2/index.html
The changes in the HTML file are appropriate. The text for the news ticker has been added inside a div with class "panel". The CSS file has been linked correctly.

#### challenge-2/style.css
The CSS file has been added with styles for the news ticker. The text is set to move with a linear animation that repeats indefinitely, which meets requirements 2 and 4.

However, there are two requirements that are not met:

1. Requirement 1: The speed of the text movement is set to 15s. This might be too fast or too slow depending on the amount of text and its size. It's hard to tell without seeing it in action, but this is something to keep in mind.

2. Requirement 3: There is no code to stop the animation while hovering over the text. This can be achieved by adding a hover state to the paragraph element with the animation-play-state property set to "paused", like so:

```css
.panel p:hover {
    animation-play-state: paused;
}
```

In conclusion, the code changes cover most of the task requirements, but there are a couple of points that need to be addressed to fully meet all the requirements.

**Review by NerdbordMentor — APPROVED** [review]
## Code Review Report

### Task goal
Develop an interactive, moving news ticker according to provided assets.

### Requirements
1. Text should be moving with speed enabling reading
2. Animation should be set to linear
3. While hovering the animation should stop
4. Animation should be looped indefinitely

### Code Changes Review

#### challenge-2/index.html
The changes in the HTML file are in line with the task requirements. The ticker text is added within paragraph tags and enclosed within a div with class "panel". The stylesheet link is also added correctly.

#### challenge-2/style.css
The CSS file is added and it contains styles that meet the task requirements. 

1. The text is set to move with the animation named "slide" which is set to take 20 seconds. This should be a sufficient speed for reading.
2. The animation is set to linear as required.
3. The animation stops while hovering over the text as specified by the rule `.panel p:hover { animation-play-state: paused; }`.
4. The animation is set to loop indefinitely with the keyword `infinite`.

In conclusion, the provided code changes cover all the task requirements.
